scalding-serialization-macros/src/main/scala/com/twitter/scalding/serialization/macros/impl/ordered_serialization/TreeOrderedBuf.scala

```diff
@@ -166,19 +178,25 @@ object TreeOrderedBuf {
         case m: MaybeLengthCalculation[_] =>
           val tmpLenRes = freshT("tmpLenRes")
           q"""
-            @inline def noLenCalc = {
-              $noLenCalc
-            }
             @inline def withLenCalc(cnt: Int) = {
```

Comment:
why not manually inline this be doing `${withLenCalc(argTree)` here? Can you add a comment about this choice?